src/main/java/appeng/me/storage/ItemCellInventory.java

```diff
@@ -95,7 +90,7 @@ public static boolean isCell( final ItemStack i )
 		final Item type = i.getItem();
 		if( type instanceof IStorageCell )
 		{
-			if ( ( (IStorageCell) type ).getChannel() == AEApi.instance().storage().getStorageChannel( IItemStorageChannel.class ) )
+			if( ( (IStorageCell) type ).getChannel() == AEApi.instance().storage().getStorageChannel( IItemStorageChannel.class ) )
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Merge this if statement with the enclosing one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1066)